syntax = "proto3";

message Dice {
  int32 rolledNumber = 1;
  DiceType type = 2;
}

message DiceType { 
	enum Type {
	  D4 = 0;
	  D6 = 1;
	  D8 = 2;	
	  D10 = 3;
	  D12 = 4;
	  D20 = 5;
	  D100 = 6;
	}
}